internal/server/grpc_test.go

```diff
@@ -84,7 +84,7 @@ func (s *GRPCServerTestSuite) TestFetchShouldReturnOneItem() {
 
 func (s *GRPCServerTestSuite) TestFetchShouldReturnTwoItems() {
 	broker := &mockBroker{}
-	broker.On("Delivery", mock.Anything).Once().Return(s.makeChannel("hello", "salam"), nil)
+	broker.On("Delivery", mock.Anything).Return(s.makeChannel("hello", "salam"), nil)
```

Comment:
It's ok to call `Delivery()` multiple time. This method returns `ErrClosed` and there isnt another way to capture this error. Checking delivery channel (which is closed or not) isnt good alternative to check broker closed are not (Its treaky and unexpected).